### PR TITLE
added PedroTechnologies link

### LINKED ISSUE #784: [Add] <Web Development Youtube resources>
### Site URL

https://www.youtube.com/@PedroTechnologies

### Description of the above provided site

At PedroTechnologies Videos about Web Development! ReactJS, NodeJS, MySQL, Express, MongoDB, GraphQL and much more!

### Category

Youtube

### Sub Category

"Web Development"

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by rony0000013**
## Fixes Issue

Added 1 resource in the youtube section under Web Development (PedroTechnologies).

This closes #784

## Screenshots

2023-05-29 23 18 24 localhost eed513408538

### CODE REVIEW

File: database/youtube/web-development.json [hunk 1]
```diff
@@ -182,5 +182,13 @@
     "category": "youtube",
     "subcategory": "web-development",
     "language": "hindi"
+  },
+  {
+    "name": "PedroTechnologies",
+    "description": "Welcome to PedroTechnologies' captivating world of Web Development! Explore a treasure trove of educational videos covering ReactJS, NodeJS, MySQL, Express, MongoDB, GraphQL, and more.",
```
Comment (CBID2) [inline database/youtube/web-development.json:188]: [Proofreading]
```suggestion
    "description": "Welcome to PedroTechnologies' captivating world of Web Development! Explore a treasure trove of educational videos covering ReactJS, NodeJS, MySQL, Express, MongoDB, GraphQL, and more!",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @rony0000013! :) I pointed out a proofreading error that could be improved.

**Review by CBID2 — APPROVED** [review]
Great job @rony0000013! 😄 LGTM! 😄

**Review by rupali-codes — APPROVED** [review]
looks good

**Pull request merged**